docs/guide/custom-provider-resources.md

```diff
@@ -28,6 +32,9 @@ resources:
       Type: ResourceType
       Properties:
         Key: Value
+  Outputs:
+    CustomOutput:
+      Value: "My Custom Output"
```

Comment:
Would add a Description here as well. Its optional, but if people see it in the docs they know its available